### PR TITLE
Add bing AutoSuggest API

### PR DESCRIPTION
**Pull request by gu-wei**
Add bing AutoSuggest API:
1. generated stub code from spec file 
2. add wrapper code and proj files
3. add test code

### CODE REVIEW

**Comment by gu-wei** [comment]
@azuresdkci, do you know who should I ping to review the PR?

**Comment by dsgouda** [comment]
@gu-wei Please complete the CLA process here
https://cla.opensource.microsoft.com/
This should kick off the CI builds after which we can review the PR.
Apologize for the delay

**Comment by gu-wei** [comment]
@dsgouda 
I've clicked the "**Sign In**" button on page https://cla.opensource.microsoft.com, it redirected to the same page and nothing happened.:(

**Comment by dsgouda** [comment]
@shahabhijeet can you take a look

**Comment by dsgouda** [comment]
@gu-wei Can you link the related REST spec PR
and start an email thread with @shahabhijeet and I.

**Comment by gu-wei** [comment]
@dsgouda , @shahabhijeet 
here is the related spec PR: https://github.com/Azure/azure-rest-api-specs/pull/3180

File: src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest/Microsoft.Azure.CognitiveServices.Search.AutoSuggest.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search.AutoSuggest</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services AutoSuggest API.</Description>
+    <VersionPrefix>1.2.0</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest/Microsoft.Azure.CognitiveServices.Search.AutoSuggest.csproj]: Is this is a preview package or a stable package.
Versioning must start from `0.9.0-preview` for preview versions and `1.0.0` for stable versions

File: src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest/generate.cmd [hunk 2]
```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\..\..\..\tools\generate.cmd cognitiveservices/data-plane/AutoSuggest %*
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest/generate.cmd]: Please replace this with a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/generate.ps1) and regenerate the code.

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest.Tests/Microsoft.Azure.CognitiveServices.Search.AutoSuggest.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft.Azure.CognitiveServices.Search.AutoSuggest.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Search.AutoSuggest.Tests</AssemblyName>
+    <VersionPrefix>1.1.0-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest.Tests/Microsoft.Azure.CognitiveServices.Search.AutoSuggest.Tests.csproj]: Test packages are never published and must always be set to version `1.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please run `msbuild build.proj /p:Scope=SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest`
and commit any artifacts generated.

**Comment by dsgouda** [comment]
@gu-wei PR still has 6 commits, if you could merge them into a single one, we are good to go here.

**Pull request closed without merging**